gxformat2/markdown_parse.py

```diff
@@ -59,12 +59,17 @@ def validate_galaxy_markdown(galaxy_markdown, internal=True):
 
         def invalid_line(template, **kwd):
             if "line" in kwd:
-                kwd["line"] = line.rstrip("\r\n")
```

Comment:
This code is replicated in Galaxy - did you pull those changes in?

Reply:
Hmm, the Galaxy code doesn't look like it'd pass, but I can try.